src/cftlib/java/uk/gov/hmcts/ethos/replacement/docmosis/CftlibConfig.java

```diff
@@ -121,7 +124,7 @@ public void configure(CFTLib lib) {
 
     private void createRoles(CFTLib lib) {
         lib.createRoles(
-                "caseworker-employment",
+            CASEWORKER_EMPLOYMENT,
                 "caseworker-employment-api",
```

Comment:
Why not have a static value for all of these roles?

Reply:
Done